### ISSUE TITLE
Optionally highlight .css files

### ISSUE THREAD

**Issue by jaydenseric** [azat-io/atom-language-postcss#7]
An option to highlight `.css` files would be a nice enhancement.

**Comment by azat-io** [comment]
Yes, I think so too. And maybe `.pcss` extension will be used for [SugarSS](https://github.com/postcss/sugarss).
We'll fix that.

**Comment by nulltier** [comment]
With extensions *.pcss and *.postcss also exists a problem with autocomplete-css, It doesn't work in files with such extensions.
